install/kubernetes/helm/istio/charts/sidecarInjectorWebhook/values.yaml

```diff
@@ -5,6 +5,12 @@ enabled: true
 replicaCount: 1
 image: sidecar_injector
 enableNamespacesByDefault: false
+resources:
+  requests:
+    cpu: 10m
+    memory: 128Mi
+  limits:
+    memory: 128Mi
```

Comment:
this seems low. I don't expect the injector to use more than this, but if its using 200mb of ram its not going to DOS the rest of the node. It seems like we could make this at least 256mb to be safe.

Maybe I am being overly conservative here though